feature/gnoi/factory_reset/factory_reset_test.go

```diff
@@ -0,0 +1,206 @@
+package factory_reset
+
+import (
+	"context"
+	"fmt"
+	"path"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	frpb "github.com/openconfig/gnoi/factory_reset"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/testt"
+)
+
+var (
+	filesCreated      = []string{}
+	fileCreateDevRand = "bash  dd if=/dev/urandom of=%s bs=1M count=2"
+	checkFileExists   = "bash [ -f \"%s\" ] && echo \"YES_exists\""
+	fileExists        = "YES_exists"
+	fileCreate        = "bash fallocate -l %dM %s"
+)
+
+const maxRebootTime = 40 // 40 mins wait time for the factory reset and sztp to kick in
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+type encryptionCommands struct {
+	EncrytionStatus      string
```

Comment:
"Encrytion" -- should it be "Encryption"?
Also, this word is used a lot in code, search and change every word please